src/commands/deployWorkspaceProject/ShouldSaveDeploySettingsPromptStep.ts

```diff
@@ -20,7 +20,7 @@ export class ShouldSaveDeploySettingsPromptStep extends AzureWizardPromptStep<De
             return;
         }
 
-        const saveOrOverwrite: string = settings ? localize('overwrite', 'overwrite') : localize('save', 'save');
+        const saveOrOverwrite: string = hasNoDeployWorkspaceProjectSettings(settings) ? localize('save', 'save') : localize('overwrite', 'overwrite');
         const saveItem = { title: localize('saveItem', 'Save...') };
```

Comment:
Out of curiosity, what does this modal look like? I'd imagine two buttons with an ellipsis would look wonky.

Reply:
![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/23febebd-5679-4218-a3e2-9d999197dc38)

Reply:
I'm happy to remove the ellipses too

Reply:
Yeah, I'd remove them. That ellipses rule is usually for commands to notify the user that there is going to be a follow up prompt. I think for a button (even if there are follow up actions) it looks way too unnatural (and kind of passive aggressive).

Reply:
Yeah, I'd remove them... That ellipses rule is usually for commands to notify the user that there is going to be a follow up prompt... I think for a button (even if there are follow up actions) it looks way too unnatural (and kind of passive aggressive)...

For example, feels way worse.